### DISCUSSION TITLE
Welcome to tw3-resources-perks-overhaul Discussions!

### DISCUSSION THREAD

**Discussion by Aelto** [Aelto/tw3-resources-perks-overhaul#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by minhSilver** [comment]
Hi there!
I would like to report some bugs (although I'm not entirely sure if it;s my fault lol):
Heavy Attack DOES NOT DRAIN MY STAMINA (but Light does, and interrupts regeneration as intended).
Also, it seems adjusting the attack speed on stamina loss also affect the whole combat animation at full stamina. My animations start out noticeably slow, until I set them values by default.
Can you help looking at those? Thanks.

**Reply by Aelto** [reply]
Hello!

There is a perk that removes the stamina cost from heavy attacks, this is probably why they don't cost anything. You can read about it [here](https://github.com/Aelto/tw3-resources-perks-overhaul#dps-tree)

As for the speed, i just looked at the formula and there is an issue with it indeed. I made this [issue](https://github.com/Aelto/tw3-resources-perks-overhaul/issues/2) that will track the updates i will do to it. 

Thanks for the report!

**Reply by Aelto** [reply]
Alright it's fixed, you can get the new version [here](https://github.com/Aelto/tw3-resources-perks-overhaul/releases/tag/v1.6.1) ! Thanks for the report!

**Reply by minhSilver** [reply]
Thanks for your quick response haha. I will try it out.
But hey, I still think there is a small problem with the heavy attack perk **_at the start of battle_**: If I can still perform H.Attacks, but at no stamina loss, and also no adrenaline loss, does that mean I get maximum damage _for almost free_?

**Reply by Aelto** [reply]
Yes! This is intended. If you compare for the same perk in the tank branch you'll notice the other perk mentions that the damage scale with adrenaline while the perk in the DPS branch doesn't mention it.

The main goal of RPO is to encourage you to play differently, or at least not to spam fast attacks. If you are thinking about abusing the heavy attacks, then RPO did its job. This is all done on purpose, heavy attacks are powerful and with this perk can be executed for free, but they are also risky to use as they are a lot slower. On top of that, adrenaline is an important ressource, especially with RPO. And depending on your build you may sometimes want to avoid using heavy attacks to keep your adrenaline for a free sign cast for example or for a free dodge.

This is why i made this mod, combat is now an interactive experience where you will have to actively think about what you'e doing in order to play at your full potential. However, the mod is still friendly enough that it doesn't come in the way if you don't want to do it !

**Reply by minhSilver** [reply]
Ohhhh I get it now ^^

**Comment by ProRock7329** [comment]
Is it compatible with Next Gen 4.02h? Why don't you resume mod support on nexus? I hope you are still working on adding lines?) We really need a completed version! And to make it convenient to download. GitHub sucks
